scripts/backend_test_shards.json

```diff
@@ -192,7 +192,8 @@
     ],
     "9": [
         "core.domain.user_jobs_one_off_test",
-        "core.domain.topic_validators_test"
+        "core.domain.topic_validators_test",
+        "core.domain.exploration_validators_test"
```

Comment:
With this addition, shard 9 is taking 33 mins to run, which is longer than our goal of 30 mins. Could you add this to shard 4 instead, which currently takes only 13 mins?

![Screen Shot 2021-02-16 at 21 30 46](https://user-images.githubusercontent.com/19878639/108148213-5d74a000-709e-11eb-81d3-0ebb95ef7fe3.png)

In general, you should add new tests to the shard with the fastest run-time. This guidance is on the [backend tests wiki page](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests) in case you forget.

Reply:
@U8NWXD Thanks! Will keep this in mind. I have made changes.

Reply:
Done